front/src/js/activity_bar.js

```diff
@@ -1,11 +1,19 @@
 const activityBar = {
     name: "activity-bar",
     props: ["currentRoom", "isWriting"],
+    methods:{
+        share: function(currentRoom) { 
+            id = this.currentRoom._id.$oid
```

Comment:
I think $iod was removed from the API responses it's now simply .id, @Draeggon can you confirm ?

Reply:
yep but @js-all will fix it in his pr that is coming soon

Reply:
and it would be
```js
this.currentRoom.id
```

Reply:
yes in #151. The feature will come in front with #179